gnucash/import-export/ofx/gnc-ofx-import.cpp

```diff
@@ -438,93 +460,56 @@ fill_transaction_description(Transaction *transaction, OfxTransactionData *data)
 {
     /* Put transaction name in Description, or memo if name unavailable */
     if (data->name_valid)
-    {
         xaccTransSetDescription(transaction, data->name);
-    }
     else if (data->memo_valid)
-    {
         xaccTransSetDescription(transaction, data->memo);
-    }
 }
 
 static void
 fill_transaction_notes(Transaction *transaction, OfxTransactionData *data)
 {
     /* Put everything else in the Notes field */
-    char *notes = g_strdup_printf("OFX ext. info: ");
+    std::ostringstream notes;
+    notes << "OFX ext. info: ";
 
     if (data->transactiontype_valid)
-    {
-        char *tmp = notes;
-        notes = g_strdup_printf("%s%s%s", tmp, "|Trans type:",
-                                gnc_ofx_ttype_to_string(data->transactiontype));
-        g_free(tmp);
-    }
+        notes << "|Trans type:" << gnc_ofx_ttype_to_string(data->transactiontype);
 
     if (data->invtransactiontype_valid)
-    {
-        char *tmp = notes;
-        notes = g_strdup_printf("%s%s%s", tmp, "|Investment Trans type:",
-                                gnc_ofx_invttype_to_str(data->invtransactiontype));
-        g_free(tmp);
-    }
-    if (data->memo_valid && data->name_valid) /* Copy only if memo wasn't put in Description */
-    {
-        char *tmp = notes;
-        notes = g_strdup_printf("%s%s%s", tmp, "|Memo:", data->memo);
-        g_free(tmp);
-    }
+        notes << "|Investment Trans type:" << gnc_ofx_invttype_to_str (data->invtransactiontype);
+
+    /* Copy only if memo wasn't put in Description */
+    if (data->memo_valid && data->name_valid)
+        notes << "|Memo:" << data->memo;
+
     if (data->date_funds_available_valid)
     {
         char dest_string[MAX_DATE_LENGTH];
-        time64 time = data->date_funds_available;
-        char *tmp = notes;
-
-        gnc_time64_to_iso8601_buff (time, dest_string);
-        notes = g_strdup_printf("%s%s%s", tmp,
-				"|Date funds available:", dest_string);
-        g_free(tmp);
+        gnc_time64_to_iso8601_buff (data->date_funds_available, dest_string);
+        notes << "|Date funds available:" << dest_string;
```

Comment:
Out of scope for the current PR but shouldn't these strings be translated?

Reply:
I think they should...